source/enetstuff.lua

```diff
@@ -0,0 +1,117 @@
+
+
+local EnetHander = {}
+
+local server
+local client
+
+local timerHostSendInterval = 0.05
+local timerHostSendTimer = timerHostSendInterval
+
+local timerClientSendInterval = 0.05
+local timerClientSendTimer = timerClientSendInterval
+
+function EnetHander.createHost()
+-- called by menu
+
+	server = sock.newServer(hostIPAddress, 22122)
+	
+    -- Called when receiving a message of type "connect"
+    server:on("connect", function(data, client)
+        -- Send a message of type "welcome" back to the connected client
+		client:send("welcome", client:getConnectId())
+		
+		newLander = Lander.create()
+		newLander.connectionID = client:getConnectId()
+		table.insert(garrLanders, newLander)
+	end)
+	
+	server:on("clientdata", function(landerObject, clientInfo)

```

Comment:
I changed from `landerObject` to `lander` in my code since there are no other types of lander variables.. well Lander (the module), but I think it should be fine. I prefer readability before descriptive.

Reply:
Yup. I have synced mine to yours.